### PR TITLE
Entity cache: move to preview

### PR DESCRIPTION
**Pull request by Geal**
*Description here*

Fixes #**issue_number**

---

**Checklist**

Complete the checklist (and note appropriate exceptions) before the PR is marked ready-for-review.

- [ ] Changes are compatible[^1]
- [ ] Documentation[^2] completed
- [ ] Performance impact assessed and acceptable
- Tests added and passing[^3]
    - [ ] Unit Tests
    - [ ] Integration Tests
    - [ ] Manual Tests

**Exceptions**

*Note any exceptions here*

**Notes**

[^1]: It may be appropriate to bring upcoming changes to the attention of other (impacted) groups. Please endeavour to do this before seeking PR approval. The mechanism for doing this will vary considerably, so use your judgement as to how and when to do this.
[^2]: Configuration is an important part of many changes. Where applicable please try to document configuration examples.
[^3]: Tick whichever testing boxes are applicable. If you are adding Manual Tests, please document the manual testing (extensively) in the Exceptions.

### CODE REVIEW

File: apollo-router/feature_discussions.json [hunk 1]
```diff
@@ -5,5 +5,7 @@
     "experimental_when_header": "https://github.com/apollographql/router/discussions/1961",
     "experimental_batching": "https://github.com/apollographql/router/discussions/3840"
   },
-  "preview": {}
+  "preview": {
+    "preview_entity_cache": "TBD"
```
Comment (bnjjj) [inline apollo-router/feature_discussions.json:9]: It has to be created I guess

**Review by bnjjj — COMMENTED** [review]
If you don't break anything and just rename it, should we write a small migration for this ?

File: apollo-router/feature_discussions.json (hunk 1 above)
Comment (Geal) [inline apollo-router/feature_discussions.json:9]:
```suggestion
    "preview_entity_cache": "https://github.com/apollographql/router/discussions/4592"
```

Comment (Geal) [reply apollo-router/feature_discussions.json]: done

**Review by bnjjj — APPROVED** [review]

**Comment by Geal** [comment]
merging this one now, the changeset will be in https://github.com/apollographql/router/pull/4195

**Pull request merged**